tests/Formatters/OaiDcFormatterTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Terraformers\OpenArchive\Tests\Formatters;
+
+use DOMDocument;
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\Dev\SapphireTest;
+use Terraformers\OpenArchive\Extensions\VersionedOaiRecordManager;
+use Terraformers\OpenArchive\Formatters\OaiDcFormatter;
+use Terraformers\OpenArchive\Tests\Mocks\SiteTreeExtension;
+
+class OaiDcFormatterTest extends SapphireTest
+{
+
+    protected static $fixture_file = 'OaiDcFormatterTest.yml'; // phpcs:ignore
+
+    protected static array $required_extensions = [
+        SiteTree::class => [
+            VersionedOaiRecordManager::class,
+            SiteTreeExtension::class,
+        ],
+    ];
```

Comment:
Untested, but I think something like this should get passed the linter.
```suggestion
    /*
     * @phpcsSuppress SlevomatCodingStandard.TypeHints
     */
    protected static $required_extensions = [
        SiteTree::class => [
            VersionedOaiRecordManager::class,
            SiteTreeExtension::class,
        ],
    ];
```